### ISSUE TITLE
[Help] Error appears in basic installation.

### ISSUE THREAD

**Issue by Adrek** [stalniy/casl#522]
**Describe the bug**
I'm new using CASL . When I use $can in template of Vue 3 the console shows:
https://i.imgur.com/7EEzINa.png (See image)
``
index.js:17 Uncaught (in promise) TypeError: Cannot read property 'apply' of undefined
``

**To Reproduce**
1. Create blank Vue 3 project 
2. Install @casl/ability and @casl/vue
3. Setup USE configuration with globalProperties
4. Use v-if="$can('create', 'Post')" in template

**Expected behavior**
Should show the inner text inside div

**Interactive example** 
I did it in local and CodeSanbox. The result is same
https://codesandbox.io/s/dreamy-pine-2lb1l?file=/src/components/Child.vue

**CASL Version**

`@casl/ability` - v^5.3.1
`@casl/vue` - v^2.1.0

**Environment**:
Node v14.17.1
Google Chrome v91.0.4472.124

**Comment by stalniy** [comment]
Thanks, fixed in @casl/vue@2.1.1

**Comment by Adrek** [comment]
> Thanks, fixed in @casl/vue@2.1.1

Thanks. 

I tried it on a blank project with Javascript and it works ok. The text "Should this text" appears. However I tested it on a blank project with Typescript and the text is not showing up. Or is it that I am registering it wrong?

I test in local and CodeSandbox. The result is same:
https://codesandbox.io/s/old-frost-tgwie?file=/src/defineAbility.ts


 =================================
PD: In typescript, if I want to use 'manage' and 'all' to have all permissions like tutorial , do I have to add/define them to Actions and Subjects respectively?

From this:
```
type Actions = 'create'
type Subjects = 'Post'
```

To this ?:
```
type Actions =  'manage' | 'create'
type Subjects =  'all' | 'Post'
```

**Comment by stalniy** [comment]
About typescript and actions check this example https://github.com/stalniy/casl-examples/blob/master/packages/vue-todo/src/config/ability.ts
